features/step_definitions/inspection_steps.rb

```diff
@@ -43,14 +43,14 @@
     count = 0
     while count < attempts
       break if element_exists("* {text CONTAINS[c] '#{text}'}")
-      scroll_down
+      pan_down
```

Comment:
Yes, it was breaking when using scroll with a message like "No scrollable view found" or something like that, but switching to pan worked. What exactly do you mean by broader support? I did test several other of our feature files which use this command and they worked fine with the changed implementation.